src/dex-ui/pages/Pools/Pools.tsx

```diff
@@ -195,7 +196,18 @@ const Pools = (): JSX.Element => {
                 >
                   {userPool ? "Withdraw" : "Swap"}
                 </Link>
-                <Link textDecoration={"underline"} color={"#3078FF"} as={RouterLink} to={"/pools/add-liquidity"}>
+                <Link
+                  textDecoration={"underline"}
+                  color={"#3078FF"}
+                  onClick={() => {
+                    navigate(`/pools/add-liquidity`, {
```

Comment:
Let's update this to use route parameters instead of state. That allows the user to directly navigate to the add-liquidity form for a specific pool (i.e. `/pools/add-liquidity/${pairAccountId}`): https://v5.reactrouter.com/web/api/Hooks/useparams

Reply:
Done